ui/src/app/cohort-review/page-layout/page-layout.ts

```diff
@@ -20,6 +23,20 @@ export class PageLayout implements OnInit, OnDestroy {
   reviewPresent: boolean;
   cohortLoaded = false;
   readonly = false;
+  subscription: Subscription;
+
+  constructor(private location: Location, private router: Router) {
+    this.subscription = router.events.subscribe(event => {
+      if (event instanceof NavigationStart) {
+        const oldRoute = router.url.split('/').pop();
```

Comment:
Can we determine the same conditions by possibly setting some state instead of explicitly checking the url? Feels like url/routing could change and this would be broken.

Reply:
I could potentially track the active component in state, but I'd still need to compare the urls to determine if the browser's back button was clicked.